### PR TITLE
Iceberg product tests

### PR DESCRIPTION
**Pull request by aalbu**
Depends on https://github.com/prestosql/docker-images/pull/64

### CODE REVIEW

File: presto-product-tests/src/main/resources/tempto-configuration.yaml [hunk 1]
```diff
@@ -123,6 +123,19 @@ databases:
     jdbc_pooling: true
     table_manager_type: jdbc
 
+  spark:
+      host: spark
```
Comment (findepi) [inline presto-product-tests/src/main/resources/tempto-configuration.yaml:127]: use 2 space ident for consistency with the rest of the file

(we can change in separate commit/pr)

File: presto-product-tests/src/main/resources/tempto-configuration.yaml [hunk 2]
```diff
@@ -123,6 +123,19 @@ databases:
...
+      jdbc_driver_class: org.apache.hive.jdbc.HiveDriver
+      jdbc_url: jdbc:hive2://${databases.spark.host}:10213
+      jdbc_user: hive
+      jdbc_password: na
+      jdbc_pooling: false
+      schema: default
+      prepare_statement:
+          - USE ${databases.spark.schema}
+      table_manager_type: jdbc
+      warehouse_directory_path: /user/hive/warehouse
```
Comment (findepi) [inline presto-product-tests/src/main/resources/tempto-configuration.yaml:137]: used?

Comment (aalbu) [reply presto-product-tests/src/main/resources/tempto-configuration.yaml:137]: Not at this point, I'll remove.

**Comment by electrum** [comment]
`prestodev/spark3.0-iceberg:31` is live now

**Comment by electrum** [comment]
Thanks!

**Pull request merged**